borg/archiver.py

```diff
@@ -817,6 +850,12 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--exclude-from', dest='exclude_files',
                                type=argparse.FileType('r'), action='append',
                                metavar='EXCLUDEFILE', help='read exclude patterns from EXCLUDEFILE, one per line')
+        subparser.add_argument('--exclude-regex', dest='excludes',
```

Comment:
Now or later at 1.0 we could also drop this and just have same patterns on commandline as in the file.